WooCommerce/Classes/Extensions/UIStackView+Subviews.swift

```diff
@@ -0,0 +1,15 @@
+import UIKit
+
+extension UIStackView {
+    /// Removes all the arranged subviews in the stack view.
+    ///
+    func removeAllArrangedSubviews() {
+        arrangedSubviews.forEach({ removeArrangedSubview($0) })
```

Comment:
Unimportant: I was just curious so I tried it out. It looks like you can do this magic too: 

```swift
        arrangedSubviews.forEach(removeArrangedSubview)
```

Reply:
I've seen this syntax but I think SourceKit wasn't working properly in my Xcode so it didn't get autocomplete suggestions when I tried this before. will update!